app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
   include IdmeAuthenticatable
 
+  def idme_sign_in(after_login: nil)
+    send_mixpanel_event(event_name: "click_sign_in")
+    redirect_to idme_authorize(after_login: after_login)
```

Comment:
I think this should be `user_idme_omniauth_authorize_path(after_login: params[:after_login])` to match the previous behavior? That will allow us to stub out the behavior in feature specs (which I believe is the reason they are failing right now -- they are actually trying to request the id.me OAuth flow via the feature specs web server)